tests/testthat/test-sparsevctrs.R

```diff
@@ -0,0 +1,30 @@
+test_that("recipe() accepts sparse tibbles", {
+
```

Comment:
```suggestion
  skip_if_not_installed("modeldata")

```

re:

```
══ Failed tests ════════════════════════════════════════════════════════════════
── Error ('test-sparsevctrs.R:3:3'): recipe() accepts sparse tibbles ───────────
<packageNotFoundError/error/condition>
Error in `loadNamespace(x)`: there is no package called 'modeldata'
Backtrace:
    ▆
 1. ├─recipes:::sparse_hotel_rates() at test-sparsevctrs.R:3:3
 2. └─base::loadNamespace(x) at tests/testthat/helper-sparsevctrs.R:6:3
 3.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
 4.     └─base (local) withOneRestart(expr, restarts[[1L]])
 5.       └─base (local) doWithOneRestart(return(expr), restart)

```

in the "hard" check.